support/kueue.go

```diff
@@ -65,7 +65,7 @@ func CreateKueueClusterQueue(t Test, clusterQueueSpec kueuev1beta1.ClusterQueueS
 	return clusterQueue
 }
 
-func CreateKueueLocalQueue(t Test, namespace, clusterQueueName string) *kueuev1beta1.LocalQueue {
+func CreateKueueLocalQueue(t Test, namespace, clusterQueueName string, annotations map[string]string) *kueuev1beta1.LocalQueue {
```

Comment:
I'd suggest to use a variadic argument of type `Option[*kueuev1beta1.LocalQueue]` as it is done for namespace:

https://github.com/project-codeflare/codeflare-common/blob/b41e108b77fa99de86a7ecfa92590ce00dcfae0f/support/namespace.go#L26

And provide a `DefaultLocalQueue` option.

Reply:
@astefanutti I have made changes accordingly, please review again : https://github.com/project-codeflare/codeflare-common/pull/59/commits/c290d4b049cf3799309b79837991a85aa104392b